wandb/sdk/lib/server.py

```diff
@@ -36,3 +37,10 @@ def query_with_timeout(self, timeout: int | float = 5) -> None:
         # TODO(jhr): should we kill the thread?
         self._viewer, self._serverinfo = viewer_tuple
         self._flags = json.loads(self._viewer.get("flags", "{}"))
+
+    @property
+    def viewer(self) -> dict[str, Any]:
```

Comment:
While you're here, can you document what this is conceptually, and if the `dict` follows any expected format?